README.md

```diff
@@ -1,63 +1,256 @@
-# Unikraft "hello world" Application
+# Helloworld on Unikraft
 
-This application prints a basic "Hello World!" message.
+This application starts a Helloworld application with Unikraft.
+Follow the instructions below to set up, configure, build and run Helloworld.
 
-To configure, build and run the application you need to have [kraft](https://github.com/unikraft/kraft) installed.
+## Requirements
```

Comment:
# A simple "Hello, world" application on Unikraft

To get started immediately, you can use [Unikraft's companion command-line companion tool `kraft`](https://kraftkit.sh). Simply start by running the interactive installer:
```
curl --proto '=https' --tlsv1.2 -sSf https://get.kraftkit.sh | sh
```

Once installed, simple clone this repository and run `kraft build`:
```
git clone --depth=1 https://github.com/unikraft/app-hellworld.git
cd app-helloworld;
kraft build
```

This will guide you through an interactive build process where you can select one of the available targets (architecture/platform combinations). Otherwise, we recommend building for qemu/x86_64 like so:
```
kraft build --arch x86_64 --plat qemu
```

Once built, you can instantiate the unikernel via:
```
kraft run
```

When left without any input flags, you'll be queried for the desired target architecture/platform.

## Build Unikraft from first principles (Advanced)